### ISSUE TITLE
The terminal process "C:\Users\user\.platformio\penv\Scripts\platformio.exe 'run'" terminated with exit code: 1. 

### ISSUE THREAD

**Issue by backspacestud** [SlimeVR/SlimeVR-Docs-Site#216]
Hi, I have an issue where whenever I build the project it fails and gives me exit code 1. It then lists problems it found and I'm not sure if it is something I can fix on my own at this time, is this a wide spread issue or an individual one? I followed the guide:(https://docs.slimevr.dev/firmware/configuring-project.html) with the only things that I changed being the define.h and hardcoded WIFI (i uncommented and commented these to check and the issue happened no matter what). I have tried on macOS, windows, and linux if that helps anything. Sorry for the sparce amount of info I'm not sure why I'm having this issue now and any help is appreciated, thanks.

![image](https://github.com/SlimeVR/SlimeVR-Tracker-ESP/assets/101374988/ef3b7093-3b90-40b2-ae88-5f37e330b846)
![image](https://github.com/SlimeVR/SlimeVR-Tracker-ESP/assets/101374988/efbf9a82-2ed0-4e5b-acbd-8064f196e014)

**Comment by unlogisch04** [comment]
Mybad i forgot this line:
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/src/defines.h#L35
```cpp
#define MAX_IMU_COUNT 2
```

so in total:
```cpp
#define MAX_IMU_COUNT 2
#ifndef IMU_DESC_LIST
#define IMU_DESC_LIST \
        IMU_DESC_ENTRY(IMU,        PRIMARY_IMU_ADDRESS_ONE,   IMU_ROTATION,        PIN_IMU_SCL, PIN_IMU_SDA, PIN_IMU_INT  ) \
        IMU_DESC_ENTRY(SECOND_IMU, SECONDARY_IMU_ADDRESS_TWO, SECOND_IMU_ROTATION, PIN_IMU_SCL, PIN_IMU_SDA, PIN_IMU_INT_2)
#endif
```

**Comment by backspacestud** [comment]
I didn't see that, thank you so much!

**Comment by calliePepper** [comment]
Fixed in #213 Thank you for bringing it up!
